tracer/test/snapshots/GrpcDotNet.SubmitTraces_httpclient=False.verified.txt

```diff
@@ -59,11 +59,15 @@
       aspnet_core.route: /greet.tester.greeter/streamingbothways,
       component: aspnet_core,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: POST,
       http.request.headers.host: localhost:00000,
+      http.route: /greet.tester.greeter/streamingbothways,
       http.status_code: 200,
       http.url: http://localhost:00000/greet.tester.Greeter/StreamingBothWays,
+      http.useragent: grpc-dotnet/123,
       language: dotnet,
+      network.client.ip: 127.0.0.1,
```

Comment:
Just checking, should all these tags also be on the grpc server spans? 

That question goes for the legacy grpc snapshots too, which currently don't have _any_ changes (because it bypasses aspnetcore and listens directly)

Reply:
I guess they should as long as it's over http 🤔 
Looking at the legacy ones, not sure where exactly I could add this.. maybe could leave it for a subsequent smaller PR